### DISCUSSION TITLE
{name}/RAW/QPIRI is became NAK after some Set_Command

### DISCUSSION THREAD

**Discussion by vytya** [softwarecrash/Solar2MQTT-ESP8266#129] [Q&A]
Powmr 6.2 inverter. PI30 protocol.

Noticed that after some commands RAW/QPIRI is becoming NAK.
![image](https://github.com/user-attachments/assets/a9ca08c6-09dc-4d66-8c12-33cff18add99)

Probably type of commands is not related, maybe that's some sort of async issue, like command is getting in moment of some action. 
And noticed that RAW/QPIRI is not updated until Set_Command = QPIRI is sent (waited ~10min and no update).

Is it expected?
Maybe it can be improved like if there is NAK then update ?

**Comment by softwarecrash** [comment]
Strange behavior, never Seen bevor.
Il Put it in the To-Do list for checking
